owaspSuppressions.xml

```diff
@@ -18,6 +18,11 @@
 -->
 
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.2.xsd">
+   <suppress>
+      <!-- ZooKeeper is not affected, because HttpURI is not used in our code.
```

Comment:
Shall we change this comment a bit ?

Reply:
Sure, could you please phrase it for me?